src/app/wizard/services/app-launcher-project-summary.service.ts

```diff
@@ -55,17 +56,27 @@ export class AppLauncherProjectSummaryService implements ProjectSummaryService {
    * @returns {Observable<boolean>}
    */
   setup(summary: Summary): Observable<boolean> {
-    let summaryEndPoint: string = this.END_POINT + this.API_BASE;
+    let summaryEndPoint: string = this.END_POINT +
+      (this.isTargetOpenshift(summary) ? this.API_BASE_LAUNCH : this.API_BASE_ZIP);
     return this.options.flatMap((option) => {
-      return this.http.post(summaryEndPoint, this.getPayload(summary), option)
-        .map(response => {
-          console.log(response.json());
-          return response.json();
-        })
-        .catch(this.handleError);
+      if (this.isTargetOpenshift(summary)) {
+        return this.http.post(summaryEndPoint, this.getPayload(summary), option)
+          .map(response => {
+            console.log(response.json());
+            return response.json();
+          }).catch(this.handleError);
+      } else {
+        window.open(summaryEndPoint + '?' + this.getPayload(summary));
```

Comment:
Or `window.location`, but it's probably pointing to the same object

Reply:
it's because the window.open is used in a direct action of the user, if you would delay the window.open or do it in a async way then it will be blocked.

Reply:
considering that the result is a ZIP file, I don't think that will break the navigation, I used that in the wildfly-swarm generator